torch/nn/modules/loss.py

```diff
@@ -529,8 +529,8 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
 
 
 class BCELoss(_WeightedLoss):
-    r"""Creates a criterion that measures the Binary Cross Entropy
-    between the target and the output:
+    r"""Function that measures the Binary Cross Entropy between the target and input
```

Comment:
"Function" isn't quite accurate here since it's a module- what do you think about just "Measures the..." ?

Reply:
I mistakenly copied and pasted here, updated to PR to just replace "the output" with "the input probabilities".